Paper.tex

```diff
@@ -192,6 +196,7 @@ \subsection{Which History?}
 $F_{\mathrm{London}}$            & 12965000 \\
 $F_{\mathrm{Arrow Glacier}}$     & 13773000 \\
 $F_{\mathrm{Gray Glacier}}$      & 15050000 \\
+$F_{\mathrm{Paris}}$      & 15537393 \\
```

Comment:
I believe that 15537393 was the last PoW block, while the first PoS block was 15537394.

Reply:
you're correct!

Reply:
fixed